quality/build.gradle.kts

```diff
@@ -29,7 +29,18 @@ dependencies {
 	implementation(projects.compat.agp)
 
 	compileOnly(libs.annotations.jetbrains)
-	compileOnly(libs.android.gradle)
+	compileOnly(libs.android.gradle) {
+		configurations.compileOnly {
+			// +--- com.android.tools.build:gradle:7.2.1
+			//|    +--- com.android.tools:sdk-common:30.2.1
```

Comment:
@sonatype-lift ignore https://community.sonatype.com/t/how-do-i-report-false-positives/9078

Reply:
Hi @TWiStErRob, this is certainly an interesting comment! Our tool for software component analysis will do a gradle build to determine which dependencies are used and check our database for vulnerabilities. The less sophisticated part is figuring out where the dependency is specified in the file, which uses some simple pattern matching. I'll file a bug with out team to improve the comment experience.

This is also reporting that the vulnerability is in a transitive dependency and not in `common`. You can see the vulnerability details here: https://ossindex.sonatype.org/component/pkg:maven/com.google.guava/guava@30.1-jre

Reply:
@mattjohnson while you're here, this flag seems like an interesting problem, because I didn't add any dependencies in this PR, only [removed them](https://github.com/TWiStErRob/net.twisterrob.gradle/pull/245/files#diff-12eee5795f6ef85a11ee0bd4c47b88c7d5a5b5821283111940f8969369885c4e). So the only reason this pre-existing issue (I've been depending on libs.android.gradle forever in multiple places which uses common transitively) is flagged is because suddenly lift found a place to flag it on? Or maybe more likely that it's just keying by "location in build" rather than the vulnerability / dependency?

Reply:
Correct, we build both the PR and the target branches and diff the findings. Because one finding was attributed to a line in PR and the other wasn't, it likely believed them to be distinct. Great corner case to add to our test suite 😄